tests/fixtures_mc_simulator.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+from multiprocessing import Queue
+
+from mantarray_desktop_app import MantarrayMCSimulator
+from mantarray_desktop_app import SECONDS_TO_WAIT_WHEN_POLLING_QUEUES
+import pytest
+
+
+@pytest.fixture(scope="function", name="mantarray_mc_simulator")
+def fixture_mantarray_mc_simulator():
+    input_queue = Queue()
+    testing_queue = Queue()
+    output_queue = Queue()
+    error_queue = Queue()
+    simulator = MantarrayMCSimulator(
+        input_queue,
+        output_queue,
+        error_queue,
+        testing_queue,
+        read_timeout_seconds=SECONDS_TO_WAIT_WHEN_POLLING_QUEUES,
+    )
+
+    yield input_queue, output_queue, error_queue, testing_queue, simulator
+
+
+class MantarrayMCSimulatorNoBeacons(MantarrayMCSimulator):
```

Comment:
I wonder if it'd be better to have the overridden beacon function put `bytes(0)` into the queue instead of doing absolutely nothing.  I don't think that would affect the results of reading from the queue, but it could expose unintended consequences of populating the queue.  Just trying to see how closely we can mimic the real code without overburdening the process of writing the tests